server/src/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -4932,6 +4932,32 @@ public Domain getVlanDomain(long vlanId) {
         return _networkOfferingDao.search(sc, searchFilter);
     }
 
+     @Override
+     @DB
+     public boolean releaseDomainSpecificVirtualRanges(final long domainId) {
+        final List<DomainVlanMapVO> maps = _domainVlanMapDao.listDomainVlanMapsByDomain(domainId);
+        if (maps != null && !maps.isEmpty()) {
```

Comment:
@ustcweizhou Thanks for your changes, the code is ok, just one more thing.
Could you please change the condition (**maps != null && !maps.isEmpty()**) for (**MapUtils.isEmpty(maps)**)?

Thanks